colossalai/auto_parallel/tensor_shard/utils/device/get_one_alpha_beta.py

```diff
@@ -0,0 +1,130 @@
+import math
+import socket
+import subprocess
+import time
+
+import torch
+import torch.distributed as dist
+
+MB = int((1 << 10) * 1e3)
+GB = int((1 << 20) * 1e3)
+Byte = 4
+FRAMEWORK = 20 / 1e6
+
+
+def store(time, bandwidth):
+    f = open("tmp.txt", "w")
+    f.write(str(time) + ',' + str(bandwidth))
+    f.close()
+
+
+def load():
+    f = open("tmp.txt", "r")
+    ln = f.readline().split(",")
+    f.close()
+    return (float(ln[0]), float(ln[1]))
+
+
+def execute_cmd(cmd):
+    cmd = ' '.join(cmd)
+    print(cmd)
+    subprocess.call(cmd, shell=True)
+
+
+def test(wsize, nbytes, type):
+    warmup = 5
```

Comment:
Generally we don't name a function as `test` as it is pretty vague. We should call it like `measure_xxx`.